GameWorld/ContentProject/Content/Shaders/Pbr/MetalRoughness/MetalRoughness_main.fx

```diff
@@ -56,7 +56,7 @@ GBufferMaterial GetMaterial(in PixelInputType input)
     
     if (UseMask)
     {
-        material.maskValue = MaskTexture.Sample(SampleType, texCord).a;
+        material.maskValue = MaskTexture.Sample(SampleType, texCord);
```

Comment:
Is this correct?

Reply:
Yes, it is fact correct, 
as Mask Data official are stored in R,B,G,A so you do need to real the real.
and `material.maskValue ` is `float4`